### ISSUE TITLE
Improve display of favorites for ties

### ISSUE THREAD

**Issue by JacobEvelyn** [JacobEvelyn/friends#158]
Currently, we display favorites like so:

```
1. Marie Curie              (3 activities)
2. George Washington Carver (3)
3. Elizabeth Cady Stanton   (3)
4. Someone Else             (2)
```

When in reality the top three friends are all tied for first place, so they should be displayed either like this:

```
1. Marie Curie              (3 activities)
1. George Washington Carver (3)
1. Elizabeth Cady Stanton   (3)
4. Someone Else             (2)
```

 or possibly like this:

```
*1. Marie Curie              (3 activities)
*1. George Washington Carver (3)
*1. Elizabeth Cady Stanton   (3)
 4. Someone Else             (2)
```

**Comment by JacobEvelyn** [comment]
Food for thought: http://stackoverflow.com/questions/25042260/convert-sorted-ruby-array-to-ranks-with-possible-repeats

**Comment by JacobEvelyn** [comment]
I think my leaning is for simplicity's sake to not have asterisks or any special indication for the tied numbers.

**Comment by JacobEvelyn** [comment]
This is released in v0.30.
